vignettes/datatable-benchmarking.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Benchmarking data.table"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteIndexEntry{Benchmarking data.table}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+
+<style>
+h2 {
+    font-size: 20px;
+}
+</style>
+
+This document is meant to guide on measuring performance of data.table. Single place to documents best practices or traps to avoid.  
+
+## fread: clear caches
+
+Ideally each `fread` call should be run in fresh session with the following commands preceding R execution. This clears OS cache file in RAM and HD cache.  
+
+```sh
+free -g
+sudo sh -c 'echo 3 >/proc/sys/vm/drop_caches'
+sudo lshw -class disk
+sudo hdparm -t /dev/sda
```

Comment:
could go for some explication on what these commands are doing

Reply:
General description is provided above the code chunk. User can always access manual of those unix commands.